scripts/make-dist.sh

```diff
@@ -23,5 +32,11 @@ pathtop="$package_name"
 ln -sf .. builddir/$pathtop
 rmfiles="$rmfiles builddir/$pathtop"
 trap "rm -f $rmfiles" 0
+
+# XXX(mem): In order to accept filenames with colons in it (because of a
+# version number like x.y.z:1.2.3), pass the --force-local flag to tar.
+# This is understood by GNU tar. If other tar programs (also called
```

Comment:
@jmstover - do we ever have to run `make dist` in conjunction with an OSX stuff, where bsd tar doesn't have that option?